lib/resty/acme/autossl.lua

```diff
@@ -350,6 +356,14 @@ function AUTOSSL.init_worker()
   end
   AUTOSSL.storage = storage
 
+  if not AUTOSSL.config.account_key_path then
+    local account_key = AUTOSSL.load_account_key_storage()
+    local err = AUTOSSL.client:set_account_key(AUTOSSL.client, account_key)
```

Comment:
with `:` you don't need to pass the self reference into function, we can just 
`AUTOSSL.client:set_account_key(account_key)`